crates/bevy_winit/src/lib.rs

```diff
@@ -682,7 +663,6 @@ pub fn winit_runner(mut app: App) {
                 event: DeviceEvent::MouseMotion { delta: (x, y) },
                 ..
             } => {
-                runner_state.redraw_requested = true;
```

Comment:
I believe this is a good changes, I tried to describe the issue in https://github.com/bevyengine/bevy/issues/11235#issuecomment-1879834874.

However, as far as I'm aware Bevy currently doesn't call `Window::request_redraw()` when it actually needs to redraw when using reactive mode. Keep in mind that removing this line will make the current status worse until this is properly addressed.